spec/inputs/neo4j_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+require "logstash/plugin"
+require "logstash/json"
+
+describe "inputs/neo4j" do
```

Comment:
If you are using the class or module in the describe block this will also correctly set the subject method with this value.